Gemfile

```diff
@@ -201,18 +201,20 @@ group :development, :test do
 end
 
 group :test do
+  gem 'axe-core-capybara'
```

Comment:
Only real change is addition of the two `axe-core-*` gems... the rest is just alphabetization for clarity now that this list is growing.